final_task/rss_reader/item_group.py

```diff
@@ -0,0 +1,49 @@
+import logging
+from dataclasses import dataclass
+from typing import List
+from item import Item
+from parser_rss import format_description
+from html import unescape as html_unescape
+
+
+@dataclass
+class ItemGroup:
+    feed: str
+    items: List[Item]
+
+    def __repr__(self):
+        str_item_group = 'Feed: ' + self.feed + '\n\n'
+
+        for item in self.items:
+            str_item_group += str(item) + '\n'
+
+        return str_item_group
+
+
+def get_item_group_from_feedparser(parser):
+    """ Retrieve all items from feedparser and return item group.
+
+    :type parser: 'feedparser.FeedParserDict'
+
+    :rtype: ItemGroup
+    """
+    items = list()
+
+    logging.info('Loop for retrieving items.')
+    for item in parser.entries:
+        text, img_links = format_description(item.description)
+
+        if not text:
+            continue
```

Comment:
It's better to try to avoid using continue. In this particular case it looks fine, but in more complex logic you may want to do something like:
`
if text:
    new_item = Item(
...
`
And you will not need to use continue

Reply:
fixed